### DISCUSSION TITLE
Problème avec les métriques

### DISCUSSION THREAD

**Discussion by beng-git** [danymat/INSAnonym-utils#88] [Q&A, Bug report & Support] [answered]
Bonjour,

Nous sommes en train de regarder un problème avec le calcul de plusieurs métriques (voir issue : https://github.com/danymat/INSAnonym-utils/issues/86 )

Pour l'instant les métriques qui fonctionnent correctement sont dateUtil, hourUtil et utility_distance

Ne prenez pas en considération pour le moment le score des 3 autres métriques. Si aucune solution n'est trouvée, nous les désactiverons pour le résultat de la compétition.

Désolé pour le contretemps,

BN

**Comment by beng-git** [comment, marked as answer]
Bonjour,

Suite à un soucis sur la nouvelle implémentation des métriques, j'ai décidé de rollback vers l'ancien code des métriques. Ce sont les métriques implémentées sur le serveur, ce qui explique sans doute que certains d'entre vous à qui j'avais dit de tester leurs soumissions sur le serveur m'ont dit que ça fonctionnait.

Vous pourrez les trouver ici : 

- métrique POI : https://benjamin-nguyen.fr/DARC/utility_POI_nodf.py
- métrique meet : https://benjamin-nguyen.fr/DARC/utility_meet_nodf.py
- métrique tule : https://benjamin-nguyen.fr/DARC/utility_tuile_nodf.py

Cordialement,

BN

**Reply by dat-lequoc** [reply]
Bonjour, 

Le lien vers la métrique meet ne fonctionne pas. Pourriez-vous le vérifier s'il vous plaît?

Cordialement,
Dat

**Reply by beng-git** [reply]
Voilà j'ai réparé le lien.

**Comment by dat-lequoc** [comment]
> Pour l'instant les métriques qui fonctionnent correctement sont dateUtil, hourUtil et utility_distance

Bonjour, 

Pour le moment, la métrique dateUtil ne fonctionne pas, voir issue #83, pull request #85.

Cordialement,

Dat

**Comment by zinebhit** [comment]
Bonjour, 
Il nous été informé que le site allait être down hier pour maintenance .
Est-ce que la version d'aujourd'hui sur le serveur prend en considération ces 3 anciennes métriques  ? Lesquelles seront officiellement utilisées pour la compétition ?

Cordialement,
ZH

**Reply by beng-git** [reply]
Les métriques actuellement sur le serveur sont celles indiquées ici : https://github.com/danymat/INSAnonym-utils/discussions/88#discussioncomment-1621687 compte tenu du délai un peu court pour les modifs je pense qu'on va rester avec celles là.

**Comment by beng-git** [comment]
Voiles 3 autres métriques telles qu'elles sont sur le serveur actuellement.

- https://benjamin-nguyen.fr/DARC/hourUtil_serv.py 
- https://benjamin-nguyen.fr/DARC/dateUtil_serv.py
- https://benjamin-nguyen.fr/DARC/utility_distance_serv.py

**Comment by beng-git** [comment]
Bonsoir, 

en fait je me suis trompé, les scripts ne sont pas encore poussés sur le serveur. Je viens de voir avec Daniel. L'objectif est qu'il pousse une nouvelle version des scripts sur le serveur d'ici demain soir (mardi 16/11). Il vous tiendra au courant de l'avancement de cette MAJ. 

Désolé pour le dérangement, vous pouvez utiliser les scripts en local pour le moment pour calculer vos utilités, les résutats devraient à terme être les mêmes sur le serveur.

Cordialement,

BN

**Comment by beng-git** [comment]
Bonjour,

Le serveur a été relancé avec les métriques indiquées dans ce fil. Il est possible que le temps de calcul soit plus long (15-20mins) par soumission.

Pour ceux qui ont eu des soucis, pouvez-vous confirmer le bon fonctionnement des métriques sur le serveur désormais.

Cordialement,

BN

**Comment by beng-git** [comment]
Attention les métriques en téléchargement ont été mises à jour suite à une erreur (inversion des fichiers anonyme et original sauf dans distance).
